Scripts/Motions/Graphics/AlphaToMotion.cs

```diff
@@ -0,0 +1,41 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace FlowEnt.Motions.GraphicMotions
+{
+    public class AlphaToMotion : AbstractMotion<Graphic>
+    {
+        public AlphaToMotion(Graphic item, float to) : base(item)
+        {
+            Color col = item.color;
```

Comment:
don't get the colour in the constructor. The motion might start at a later time with a different colour. 
Check https://github.com/Fried-Synapse/FlowEnt/blob/main/Scripts/Motions/Renderer/AlphaToMotion.cs